packages/shared/src/components/checklist/InstallExtensionChecklistStep.tsx

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 import type { ChecklistStepProps } from '../../lib/checklist';
 import { ChecklistStep } from './ChecklistStep';
 import { Button, ButtonVariant } from '../buttons/Button';
-import BrowsersIcon from '../../../icons/browsers.svg';
+import BrowsersIcon from '../icons/Browsers/group.svg';
```

Comment:
I see that you are now using the correct SVG. We can now just delete the modified file above.